### ISSUE TITLE
AF_INET6/PF_INET6 conflicts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sleinen/snmp-session#5]
```
Conflicting AF_INET6/PF_INET6 are being pulled in, causing at least some 
warnings:

Constant subroutine SNMP_Session::AF_INET6 redefined at 
/usr/libdata/perl5/Exporter.pm line 67.
 at /usr/local/libdata/perl5/site_perl/SNMP_Session.pm line 149
Prototype mismatch: sub SNMP_Session::AF_INET6 () vs none at 
/usr/libdata/perl5/Exporter.pm line 67.
 at /usr/local/libdata/perl5/site_perl/SNMP_Session.pm line 149
Constant subroutine SNMP_Session::PF_INET6 redefined at 
/usr/libdata/perl5/Exporter.pm line 67.
 at /usr/local/libdata/perl5/site_perl/SNMP_Session.pm line 149
Prototype mismatch: sub SNMP_Session::PF_INET6 () vs none at 
/usr/libdata/perl5/Exporter.pm line 67.
 at /usr/local/libdata/perl5/site_perl/SNMP_Session.pm line 149 
[..]

Diff attached imports only the functions which are used.
```

Original issue reported on code.google.com by `datamat...@gmail.com` on 11 Jun 2010 at 5:23

Attachments:
- [SNMP_Session.Socket6-conflict.diff](https://storage.googleapis.com/google-code-attachments/snmp-session/issue-5/comment-0/SNMP_Session.Socket6-conflict.diff)

**Comment by allanjude** [comment]
fixed in #7

**Comment by sleinen** [comment]
Thanks for the reminder—closing this issue (at long last).
